app/src/main/res/values/strings.xml

```diff
@@ -4,7 +4,7 @@
     <string name="app_name">Glia SDK Widget Example App</string>
 
     <string name="settings_value_default">Default</string>
-    <string name="settings_value_default_company_name">Glia company</string>
+    <string name="settings_value_default_company_name">Init company name</string>
```

Comment:
Shall it also be changed on iOS?

Reply:
We can suggest, but the point of this is to implicitly shoiw where the value is coming from